base/docs/helpdb/Base.jl

```diff
@@ -6225,7 +6227,8 @@ nextind
 """
     >>>(x, n)
 
-Unsigned right bit shift operator.
+Unsigned right bit shift operator, `x >>> n`. The result is `x` shifted right by `n` bits,
+where `n >= 0`, filling with `0`s. This is equivalent to `(unsigned(x) >> n) % T`.
```

Comment:
good catch.

Actually, should these be moved inline? If so, would they be better in `int.jl` or `operators.jl`?